std_msgs/Header header
float32[3] left_linear_acceleration
float32[3] left_angular_velocity
float32 left_temperature
int32[19] left_tensile_data
float32[20] left_joint_angles

float32[3] right_linear_acceleration
float32[3] right_angular_velocity
float32 right_temperature
int32[19] right_tensile_data
float32[20] right_joint_angles

uint32 timestamp